content/style.css

```diff
@@ -425,21 +433,37 @@ td, th, tr {
 }
```

Comment:
Using `inline-block`  for `.downloadlinks li` fixes the issue with alignment. I suggest to use `padding: 10px 15px;` too.

Reply:
Done.